src/app/views/login/SignInController.test.js

```diff
@@ -46,59 +48,55 @@ test("Check redirect doesn't work if not authenticated", () => {
     expect(component.find("h1").text()).toBe("Sign in to your Florence account");
 });
 
-test("Does password change form appear on state change", () => {
+test("Does password change form appear on first time sign in", () => {
```

Comment:
It would be nice to have the test running with conventions so they read properly. I  also don't understand why you call this component controller? it is just UI, don't manage any business logic
for example something like that, depends what is this component actually doing. 

```
describe("SignIn", () => {
    describe("when user first time signed in ", () => {
        it("displays prompt to change password ", () => {
            ......
        });
    });

it('validates the password', () => {})

describe("when incorrect email ", () => {
it('show warning',() => {.......})

```